dap_prinz_green_jobs/analysis/ojo_analysis/process_ojo_green_measures.py

```diff
@@ -564,6 +564,10 @@ def create_agg_data(
     """
     Much like create_agg_occ_measures but more generic to aggregate by any column
     """
```

Comment:
@india-kerle please could you check the changes to this script are what we planned to do

Reply:
taking a look !